spec/lib/stripe/credit_card_remover_spec.rb

```diff
@@ -6,42 +6,44 @@
 describe Stripe::CreditCardRemover do
   let(:credit_card) { double('credit_card', gateway_customer_profile_id: 1) }
 
-  context 'Stripe customer exists' do
-    context 'and is not deleted' do
-      it 'deletes the credit card clone and the customer' do
-        customer = double('customer', deleted?: false)
-        allow(Stripe::Customer).to receive(:retrieve).and_return(customer)
+  describe "#remove", :vcr, :stripe_version do
+    context 'Stripe customer exists' do
+      context 'and is not deleted' do
+        it 'deletes the credit card clone and the customer' do
+          customer = double('customer', deleted?: false)
```

Comment:
It's usually better to use `instance_double`, it actually make sure you are not trying stub a non existing method, see: 
https://rspec.info/features/3-12/rspec-mocks/